library/src/main/java/com/github/pwittchen/reactivesensors/library/SensorNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.github.pwittchen.reactivesensors.library;
+
+/**
+ * Created by lucas34990 on 5/12/16.
+ */
+
+public class SensorNotFoundException extends Exception {
+
+    public SensorNotFoundException(String detailMessage) {
```

Comment:
Variable named `message` would be enough. Moreover, this parameter can be `final`.